app/views/billing-accounts/view.njk

```diff
@@ -0,0 +1,194 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/pagination/macro.njk" import govukPagination %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "macros/new-line-array-items.njk" import newLineArrayItems %}
+
+{% block breadcrumbs %}
+  {# Back link #}
+  {{
+    govukBackLink({
+      text: 'Go back to bills',
+      href:  '/system/licences/' + licenceId + '/bills'
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+  {# Main heading #}
+  <div class="govuk-body">
+    <span class="govuk-caption-l"> Billing account {{ accountNumber }} </span>
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-3">
+      {{ pageTitle }}
+    </h1>
+  </div>
+
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+  {# Billing account details #}
+  <div class="govuk-body">
+    {{ govukSummaryList({
+      classes: "govuk-summary-list--no-border",
+      rows: [
+        {
+          key: {
+            text: "Created date"
+          },
+          value: {
+            text: createdDate
+          }
+        },
+        {
+          key: {
+            text: "Customer file"
+          },
+          value: {
+            text: customerFile
+          }
+        } if lastUpdated,
+        {
+          key: {
+            text: "Last updated"
+          },
+          value: {
+            text: lastUpdated
+          }
+        } if lastUpdated,
+        {
+          key: {
+            text: "Address"
+          },
+          value: {
+            html: newLineArrayItems(address)
+          }
+        },
+        {
+          value: {
+            html: '<a class="govuk-button govuk-button--secondary" href="/billing-account-entry/change-address-' + billingAccountId + '/select-account">Change address</a>'
+          }
+        }
+      ]
+    }) }}
+  </div>
+
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+  {# Bill details #}
+  {% if bills.length <= 0 %}
+    <div class="govuk-body govuk-!-padding-top-3">
+      <p>No sent bills for this billing account</p>
+    </div>
+  {% elif bills.length > 0 %}
```

Comment:
It shouldn't be nessessary to use an "else if" here, we should just be able to use an "else" as there are only 2 conditions that we are checking for. The length is either zero or it isn't.
```suggestion
  {% else %}
```